qutip/solver/heom/bofin_baths.py

```diff
@@ -428,6 +611,46 @@ def _matsubara_params(self, lam, gamma, T, Nk):
 
         return ck_real, vk_real, ck_imag, vk_imag
 
+    def spectral_density(self, w):
+        """
+        Calculates the DrudeLorentz spectral density, see Eq. 15 in the BoFiN
+        paper (DOI: 10.1103/PhysRevResearch.5.013181).
```

Comment:
Could we write the actual equation here so that users don't have to search for a paper to see what the function does?

Reply:
Sure, should have done so from the beginning, I just wasn't aware of how easy it is

Reply:
The same applies to the `spectral_density` method of the underdamped class

Reply:
The same applies to the spectral_density method of the underdamped class